ubuntu_1804.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04 as build
```

Comment:
(Copying my comment from earlier in https://github.com/sourcegraph/lsif-clang/pull/78#discussion_r914640837)

> 

Question: Is there a specific reason for using Ubuntu 18.04? If we don't have code sharing between this Dockerfile and the top-level one, it'd be nice to at least have the same exact code being duplicated across this Dockerfile and the main top-level Dockerfile.
>
> For example, another PR just added libdwarf-dev and libelf-dev as dev dependencies. It wouldn't be too much work to add them here too, and I don't expect we'll be adding many more of those, but having the same exact code sequence would make changes smoother.

cc @jlisee